### ISSUE TITLE
Issue when using typescript

### ISSUE THREAD

**Issue by nerdthatnoonelikes** [jpbberry/topgg-autoposter#1]
So I'm using topgg-autoposter with typescript and when i run tsc I get the following
```
node_modules/topgg-autoposter/dist/structs/BasePoster.d.ts(4,29): error TS2304: Cannot find name 'BotStats'.
node_modules/topgg-autoposter/dist/structs/DJSPoster.d.ts(16,25): error TS2304: Cannot find name 'BotStats'.
node_modules/topgg-autoposter/dist/structs/DJSSharderPoster.d.ts(16,25): error TS2304: Cannot find name 'BotStats'.
node_modules/topgg-autoposter/dist/structs/ErisPoster.d.ts(16,25): error TS2304: Cannot find name 'BotStats'.

**Comment by darenliang** [comment]
I have encountered the same issue.

Not a fix, but topgg-autoposter is a wrapper over [@top-gg/sdk](https://www.npmjs.com/package/@top-gg/sdk), so you can use that instead.

**Comment by Mar0xy** [comment]
Same here but I just ignore the issue as it still works without issues the errors are just annoying.

**Comment by nerdthatnoonelikes** [comment]
Thank you! I'll be closing this now.

**Comment by DarylPinto** [comment]
Re-opening this as I'm unable to use `@ts-ignore` to skip the error messages. My bot cannot compile with this package until the issue is fixed.

**Comment by jpbberry** [comment]
I'll be looking into the issue soon. Seems to be an issue regarding how types are declared in use between the SDK and the autoposter.

**Comment by jpbberry** [comment]
This was fixed 1.1.7, it was down to an issue I caused in the SDK not exporting types properly. All should be solved!
